src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -44,6 +46,10 @@
     @FXML
     private StackPane personListPanelPlaceholder;
 
+    //Add on for Mangera
```

Comment:
Minor grammar mistake. Actually I'm not sure if these comments are needed at all. Are we removing them at the end?

Reply:
Yea will remove, I put them there to avoid confusion when yall navigate through the code, so yall know which chunks are added on top of AB3. (Will be deleted in future, so I leave it first)

Reply:
So should we remove them as the very last commit for our team submission? cuz after we merge then the green tab would be gone right?

Reply:
I guess we can do manual eyeballing and start removing after v1.2 (when everyone got comfortable with the code already ba)